packages/theming/atlas/scripts/build.js

```diff
@@ -41,7 +48,72 @@ async function main() {
     mkdir("-p", join(outputDir, "themesource/atlas_web_content"));
     mkdir("-p", join(outputDir, "themesource/atlas_nativemobile_content"));
 
-    await buildAndCopyAtlas(mode === "start", outputDir);
+    if (mode === "start") {
+        if (process.argv.includes("--validate-sass")) {
+            await new Promise(resolve => {
+                const watcher = chokidar
```

Comment:
Can we use anything else besides chokidar also?

Reply:
The builtin file watching support in NodeJS is somewhat limiting due to different behaviour on platforms. Chokidar solves this issue. It's a widely used library and has a good licence. Do you have any other library on your mind?

Reply:
Snyk is fine with Chokidar and Sass. So i am resolving this discussion